sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md

```diff
@@ -1,10 +1,6 @@
 # Release History
 
-## 5.21.0-beta.2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 5.20.3 (Unreleased)
```

Comment:
Hmm! Is this intentional?

Reply:
good catch, trusted the script :), thank you,  fixed.